templates/test_config/template-longevity-5gb-1h.yaml.j2

```diff
@@ -0,0 +1,34 @@
+test_duration: 120
+
+prepare_write_cmd:  "cassandra-stress write cl=QUORUM n=10485760 -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -mode cql3 native -rate threads=80 -pop seq=1..10485760 -col 'n=FIXED(10) size=FIXED(512)' -log interval=5"
```

Comment:
I think the dataset size here is 50GB.

10M keys * 5KB key size

Reply:
Fixed